### PR TITLE
Sync changes to production

### PR DESCRIPTION
**Pull request by localden**

### CODE REVIEW

File: msal-dotnet-articles/acquiring-tokens/desktop-mobile/wam.md [hunk 1]
```diff
@@ -7,44 +7,69 @@ description: "MSAL is able to call Web Account Manager (WAM), a Windows 10+ comp
...
+
+- [Microsoft.Identity.Client](https://www.nuget.org/packages/Microsoft.Identity.Client/)(i.e. MSAL)
+- [Microsoft.Identity.Client.Broker](https://www.nuget.org/packages/Microsoft.Identity.Client.Broker/)(i.e. MSAL.Broker)
+- [Microsoft.Identity.Client.Desktop](https://www.nuget.org/packages/Microsoft.Identity.Client.Desktop/)(i.e. MSAL.Desktop)
+
+Based on which platform the client application targets, different MSAL packages must be used.
 
-- add a reference to [Microsoft.Identity.Client.Broker](https://www.nuget.org/packages/Microsoft.Identity.Client.Broker) (and you can remove any reference to Microsoft.Identity.Client.Desktop)
-- instead of `.WithBroker()`, call `.WithBrokerPreview()`.
+| Framework      | MSAL only | MSAL + MSAL.Broker | MSAL + MSAL.Desktop       |
```
Comment (bgavrilMS) [inline msal-dotnet-articles/acquiring-tokens/desktop-mobile/wam.md:44]: I would remove this table.

File: msal-dotnet-articles/acquiring-tokens/desktop-mobile/wam.md [hunk 2]
```diff
@@ -7,44 +7,69 @@ description: "MSAL is able to call Web Account Manager (WAM), a Windows 10+ comp
...
 - Adds support for Proof-of-Possession tokens.
 - Fixes assembly size issues.
 
-### To enable WAM preview
+## Enabling WAM
+
+Due to platform-specific and backwards compatibility requirements, WAM implementation resides in three packages:
+
+- [Microsoft.Identity.Client](https://www.nuget.org/packages/Microsoft.Identity.Client/)(i.e. MSAL)
+- [Microsoft.Identity.Client.Broker](https://www.nuget.org/packages/Microsoft.Identity.Client.Broker/)(i.e. MSAL.Broker)
```
Comment (bgavrilMS) [inline msal-dotnet-articles/acquiring-tokens/desktop-mobile/wam.md:39]:
```suggestion
- [Microsoft.Identity.Client.Broker](https://www.nuget.org/packages/Microsoft.Identity.Client.Broker/) - WAM support
```

File: msal-dotnet-articles/acquiring-tokens/desktop-mobile/wam.md [hunk 3]
```diff
@@ -7,44 +7,69 @@ description: "MSAL is able to call Web Account Manager (WAM), a Windows 10+ comp
...
+|----------------|:---------:|:------------------:|:-------------------------:|
+| net48          | &cross;   | &check;            | &check; (not recommended) |
+| net6.0         | &cross;   | &check;            | &cross;                   |
+| net6.0-windows | &check;   | &cross;            | &cross;                   |
+| UWP            | &check;   | &cross;            | &cross;                   |
+| netcoreapp3.1  | &cross;   | &check;            | &check; (not recommended) |
 
->[!NOTE]
-> The old WAM experience is documented at [Acquire a token using WAM](/azure/active-directory/develop/scenario-desktop-acquire-token-wam) and showcases details about redirect URI, fallback experience on older Windows, Mac and Linux, etc. which remain valid.
+Because the .NET Core 3.1 target framework has reached the [end of life](/lifecycle/products/microsoft-net-and-net-core) on Dec 13, 2022, our team will no longer offer support for applications using MSAL with this specific version of .NET Core. We recommend upgrading to the latest version of the .NET platform where possible.
```
Comment (bgavrilMS) [inline msal-dotnet-articles/acquiring-tokens/desktop-mobile/wam.md:52]: Delete mentions of netcore 3.1

File: msal-dotnet-articles/acquiring-tokens/desktop-mobile/wam.md [hunk 4]
```diff
@@ -7,44 +7,69 @@ description: "MSAL is able to call Web Account Manager (WAM), a Windows 10+ comp
 
 MSAL is able to call Web Account Manager (WAM), a Windows 10+ component that ships with the OS. This component acts as an authentication broker allowing the users of your app benefit from integration with accounts known to Windows, such as the account you signed into your Windows session.
 
-## New WAM Preview in MSAL 4.44+
```
Comment (bgavrilMS) [inline msal-dotnet-articles/acquiring-tokens/desktop-mobile/wam.md:10]: I would delete this page and keep only https://learn.microsoft.com/en-us/azure/active-directory/develop/scenario-desktop-acquire-token-wam (for which I have a PR to update it)

**Review by bgavrilMS — APPROVED** [review]

**Review by Dickson-Mwendia — APPROVED** [review]
Great stuff @localden, @jmprieur. Glad to see identity.web docs here. Not a blocker for merging, but I'd recommend removing `articles` from the `microsoft-identity-web-articles` directory. 

Aim to change the docs URLs from `https://review.learn.microsoft.com/en-us/entra/msal/dotnet/microsoft-identity-web-articles/` to `https://review.learn.microsoft.com/en-us/entra/msal/dotnet/microsoft-identity-web/`

Otherwise, it all LGTM!

File: msal-dotnet-articles/acquiring-tokens/desktop-mobile/wam.md (hunk 4 above)
Comment (localden) [reply msal-dotnet-articles/acquiring-tokens/desktop-mobile/wam.md:10]: @bgavrilMS let's take this into a new pull request.

**Pull request merged**